src/main/java/pl/marcin/project/runners/WebAppRunner.java

```diff
@@ -20,18 +24,32 @@ public class WebAppRunner implements AppRunner {
     public void runApplication() {
         ConfigurableApplicationContext context = SpringApplication.run(ProjectApplication.class);
         var customerService = context.getBean(CustomerEntityService.class);
+        var mapService = context.getBean(GeoSerwis.class);
+        try {
+            findLocation(mapService);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
we may rethink if we want to throw a new exception here. Would just logging be enough maybe?